lib/routes.js

```diff
@@ -0,0 +1,144 @@
+'use strict'
+
+const path = require('path')
+const fastifyStatic = require('@fastify/static')
+
+// URI prefix to separate static assets for swagger UI
+const staticPrefix = '/static'
+
+function getRedirectPathForTheRootRoute (url) {
+  let redirectPath
+
+  if (url.substr(-1) === '/') {
+    redirectPath = `.${staticPrefix}/index.html`
+  } else {
+    const urlPathParts = url.split('/')
+    redirectPath = `./${urlPathParts[urlPathParts.length - 1]}${staticPrefix}/index.html`
+  }
+
+  return redirectPath
+}
+
+function fastifySwagger (fastify, opts, done) {
+  let staticCSP = false
+  if (opts.staticCSP === true) {
+    const csp = require('../dist/csp.json')
+    staticCSP = `default-src 'self'; base-uri 'self'; block-all-mixed-content; font-src 'self' https: data:; frame-ancestors 'self'; img-src 'self' data: validator.swagger.io; object-src 'none'; script-src 'self' ${csp.script.join(' ')}; script-src-attr 'none'; style-src 'self' https: ${csp.style.join(' ')}; upgrade-insecure-requests;`
```

Comment:
```suggestion
    staticCSP = `default-src 'self'; base-uri 'self'; font-src 'self' https: data:; frame-ancestors 'self'; img-src 'self' data: validator.swagger.io; object-src 'none'; script-src 'self' ${csp.script.join(' ')}; script-src-attr 'none'; style-src 'self' https: ${csp.style.join(' ')}; upgrade-insecure-requests;`
```

`block-all-mixed-content` directive is deprecated, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/block-all-mixed-content.

Can probably drop unless we're targeting IE... 5?